duckduckgo/macos-browser
# Update Scheduler

Task/Issue URL: https://app.asana.com/0/1203581873609357/1205429018807263/f

**Description**:
* Start operations when the browser starts
* Start operations when the user clicks on scan

**Steps to test this PR**:
1. After creating a profile, check if we all the queued operations right away
2. If we have a valid profile, check if we run all queued operations right away when starting the browser
